pkg/chain/celo/celoutil/celoutil.go

```diff
@@ -19,10 +19,16 @@ import (
 
 var logger = log.Logger("keep-celoutil")
 
-// CeloClient wraps the core `bind.ContractBackend` interface with
+// ABI for errors bubbled out from revert calls. Not used directly as errors are
+// neither encoded strictly as method calls nor strictly as return values, nor
+// strictly as events, but some various bits of it are used for unpacking the
+// errors. See ResolveError below.
+const errorABIString = "[{\"constant\":true,\"outputs\":[{\"type\":\"string\"}],\"inputs\":[{\"name\":\"message\", \"type\":\"string\"}],\"name\":\"Error\", \"type\": \"function\"}]"
+
+// HostChainClient wraps the core `bind.ContractBackend` interface with
 // some other interfaces allowing to expose additional methods provided
 // by client implementations.
-type CeloClient interface {
+type HostChainClient interface {
```

Comment:
What do you think about naming it just `Client` (or `ChainClient`)? From the perspective of `celoutil` package, the host chain is Celo so `HostChain*` feels clumsy.

Reply:
https://github.com/keep-network/keep-common/commit/2eae82a2416dc5916848986c9a55094c9999d6e8